unittest/sql/open_address_hash-t.cc

```diff
@@ -64,8 +69,14 @@ int main(int argc __attribute__((unused)),char *argv[])
   hashie.insert(data[0]+5);
   ok(hashie.size() == 3, "wrong size, %u", hashie.size());
   // collision
+  hashie.insert(data[1] + 1); 
+  found= hashie.find(data[1] + 1);
```

Comment:
no, `found` was supposed to be the old one. How does it work now, if it inserts the same element twice?